pipeline/extract_epic.py

```diff
@@ -100,16 +100,27 @@ def get_platform_ids(game_tags: list[str]) -> list[int]:
     return platform_ids
 
 
+def remove_platforms_from_tags(game_tags: list[str]) -> list[str]:
```

Comment:
Good use of type hints in this function; makes it very clear what its utility is